### DISCUSSION TITLE
Form feed character should be treated as a newline.

### DISCUSSION THREAD

**Discussion by Festerdam** [godotengine/godot-proposals#4654] [Scripting]
The form feed character (0xc in ASCII) [is sometimes used](https://www.gnu.org/prep/standards/html_node/Formatting.html) to divide source code into logical units. Emacs allows to easily jump between them with the shortcuts C-[ and C-]. Unfortunately, as of 3.4.2, in GDScript, if one attempts to insert a form feed character into a file, one gets an error like this:
```
SCRIPT ERROR: Parse Error: Parse error: Unknown character
          at: GDScript::reload (res://src/common/Network.gd:55)
```
Due to the fact that it would positively affect anyone who uses these characters, the fact that it doesn't affect in any way people who don't use them, and the fact that it is an easy thing to implement, I propose that these characters be treated the same as newlines in GDScript source code.

**Comment by Calinou** [comment]
Feel free to open a pull request for this :slightly_smiling_face:
